test/.eslintrc

```diff
@@ -18,5 +18,8 @@
         "mocha/prefer-arrow-callback": ["error", {
             "allowNamedFunctions": true
         }]
+    },
+    "parserOptions": {
+        "ecmaVersion": 11
```

Comment:
Just wondering, why do we need this?

Reply:
The root level eslintrc overrides ecmaVersion setting to be 8. This one in the test folder extends it so it has 8 too. However bigint literals like 1n requires ecmaVersion 11 or above. Normally in test eslint config, we set env to es2020 that automatically sets the ecmaVersion to 11, but since the root level config overrides it, test eslint config needs this additional override.